tools/config/__init__.py

```diff
@@ -567,14 +583,25 @@ def _generate_booloader_build(self, target_overrides, rom_start, rom_size):
         if start > rom_size:
             raise ConfigException("Not enough memory on device to fit all "
                                   "application regions")
+    @staticmethod
+    def _align_on_sector(address, sectors):
+        target_size = -1
+        target_start = -1
+        for (start, size) in sectors:
+            if address < start:
+                break
+            target_start = start
+            target_size = size
+        sector_num = (address-target_start)//target_size
```

Comment:
This line should be `(address - target_start) / target_size` (style updated)

Reply:
It is intentionally a floor division, but I will correct the spacing.